cypress/support/commands/commands.ts

```diff
@@ -98,3 +98,6 @@ Cypress.Commands.add('shouldHaveCssVar', { prevSubject: true }, (subject, styleN
       .should('eq', evaluatedStyle);
   });
 });
+Cypress.Commands.add('hideElementBySelector', (selector:string) => {
```

Comment:
Clever workaround. But is the counter hidden in this spec related to the repositories or is it a different area/objects?

Reply:
is present in more pages as well.